clang/unittests/Format/FormatTest.cpp

```diff
@@ -29053,6 +29053,11 @@ TEST_F(FormatTest, KeywordedFunctionLikeMacros) {
                Style);
 }
 
+TEST_F(FormatTest, UnbalancedAngleBrackets) {
+  verifyNoCrash("typename foo<bar>::value, const String &>::type f();",
+                getLLVMStyleWithColumns(50));
```

Comment:
Do we need the column limit?

Reply:
Yep! Otherwise, the assertion in `ContinuationIndenter::mustBreak` wouldn't fire.